docs_outdoornav_user_manual/faq.mdx

```diff
@@ -33,52 +33,161 @@ toc_max_heading_level: 4
+3.  **What version of ROS is compatible with OutdoorNav?**
 
-    Make sure the **Edit Missions** toggle is enabled, then click the
-    **Waypoint Mode** button. Now you can click the Waypoint you wish to
-    delete. A drop down list will appear. Select **Delete**.
+    Currently OutdoorNav is developed in ROS1 (and is built in DOcker containers on top of ROS1 Noetic).
+    So, the recommended ROS version is ROS1 Noetic. However, although not recommended, it is still
+    possible to use ROS1 Melodic. Some issues may arise related to the mismatched message types.
+    Also, if trying to run any python scripts as Melodic targets Python2, whereas Noetic targets Python3.
 
-4.  **How do I add a Task to a Waypoint?**
+    Finally, official ROS2 compatibility is currently unsupported. In theory, if you only have ROS2 installed
+    on your system, you can try to install ROS1 Noetic as well and set up a ros1_to_ros2 bridge. Since the 
+    OutdoorNav software is packaged in Dockers, it will communicate directly ROS1 and then the data can be
+    redirected by the ros1_to_ros2 bridge to ROS2.
 
-    See [Add Task to Waypoint](./web_user_interface/ui_autonomous_mode#add-task) for information on how
-    to add a Task to a Waypoint.
+4.  **How can I record ROS data?**
 
-5.  **Why is the "Save Image" Task failing?**
+    There are two ways to record ROS data. The first is [through the UI](./features/rosbag_recorder) and the
+    second is the more traditional method of accessing the command line of the UGVs PC and running 
+    [rosbag record commands](http://wiki.ros.org/rosbag/Commandline).
 
-    Check the Waypoints that have a **Save Image** task in the Waypoint panel. 
-    If you see a red warning triangle beside the **Save Image** task it means 
-    that the settings for this task were not set. Set them and rerun the
-    mission.
+5.  **Where do all the video/audio/images get saved from the mission tasks?**
+
+    All of the data that gets saved (ROSbags, video recordings, audio recordings, and saved images) get stored on
+    UGVs PC at the following location: `~/clearpath_files/...`
+
+## Autonomy
+
+1.  **Am I able to send the UGV on a repeated loop?**
+
+    When generating a single mission on the UI or through the API, it is not possible for the
+    mission to start at the location of the robot and end at the location of the robot. This has to do with
+    the navigation thinking that the UGV has already arrived at its goal and will not generate a path through
+    the Waypoints. However, it is possible to split the mission into two missions (ie. one to go to a 
+    location and the the scond to return to the starting point). With these two missions you will then either be
+    able to send each mission one after the other or you can use the [Mission Scheduler](./features/mission_scheduler).
+    Add the two missions to the schedule and you will have the ability to also repeat this loop as many times as needed.

```

Comment:
I've found it easier to create a single mission with a "wait" task on the start waypoint and have a 1 metre gap between the start point and end point. Would we want to describe that approach instead or is it more confusing to the customer?

Reply:
i think it depends on if they want the loop to be fully closed or not. If a fully closed mission is required, starting and stopping at the exact same spot, then two missions are required.

Reply:
Ok, let's leave it the way you have it.